.github/workflows/deploy-to-wordpress.yml

```diff
@@ -0,0 +1,91 @@
+name: Deploy to WordPress.org
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - develop
+
+jobs:
+  release:
+    name: New release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build
+        run: npm run build
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.0
+          extensions: mbstring, intl
+          tools: composer:v2
+          coverage: none
+
+      - name: Install PHP dependencies
+        uses: ramsey/composer-install@v2
+        with:
+          composer-options: "--no-dev --optimize-autoloader"
+
+      - name: Get the version
+        id: packagejson
+        run: |
+          echo "::set-output name=version::$(jq -r '.version' package.json)"
```

Comment:
🤔 Could you elaborate on why we are getting the version from the `package.json`? I'd assume this is not changed every time, no?